packages/components/src/table/style.scss

```
@@ -290,6 +290,13 @@ th.woocommerce-table__item {
 	margin-left: $gap-smaller;
 	margin-right: $gap-smaller;
 
+	.is-placeholder {
+		@include placeholder();
+		display: inline-block;
+		height: 16px;
+		width: 200px;
```

Preferably store this in a meaningful variable at the top and use it here.

---

Sorry for not being elaborative before:

declare a variable at the top of this change, eg:

`$place-holder-width: 200px;`
```suggestion
		width: $place-holder-width;
```

---

Apologies for not being elaborative before:

you can declare a variable like this at the top of this change: 
`$place-holder-width: 200px;`
```suggestion
		width: $place-holder-width;
```

---

Gotcha. I do not see this value is used in other places so I don't use it. 
But this is good anyway. Addressed: 395006604e2f8f729c57064463604f4cf2899a46